### PR TITLE
OM-730 | Add support for login screen

### PR DESCRIPTION
**Pull request by Jokinen**
After this commit the login page more or less support the helsinki theme.

~~Some things are still missing, for instance error messages. I'll implement these once they are specced.~~ Actually found error styles and will implement now.

Screenshot 2020-04-27 at 11 16 18

### CODE REVIEW

File: sass/custom.scss [hunk 1]
```diff
@@ -5,10 +5,179 @@
...
+// Helsinki logo
+.helsinki-logo {
+  height: 44px;
+  width: 95px;
+  // This margin is 8px. the design has 6px, but it no such spacing
+  // exists
+  margin-bottom: $spacing-2xs;
+}
+
+// Input
```
Comment (badrange) [inline sass/custom.scss:87]: Nice comment!

File: sass/custom.scss [hunk 2]
```diff
@@ -5,10 +5,179 @@
...
+// Input label
+// HDS provides styles for the label element, but they are out of sync
+// with the definitions in the design side. These overrides fill the
+// gaps.
+.hds-text-input__label {
+  font-size: $font-size-body-small;
+}
+
+// Button
+// The styles for HDS button seem are not precise enough to reproduce
```
Comment (badrange) [inline sass/custom.scss:123]: Have you mentioned this issue "upstream" to the HDS developers?

File: helsinki/login/template.ftl [hunk 3]
```diff
@@ -61,10 +61,12 @@
                         <span class="subtitle"><span class="required">*</span> ${msg("requiredFields")}</span>
                     </div>
                     <div class="col-md-10">
+                        <img class="${properties.hsLogo!}" src="${url.resourcesPath}/img/helsinki-logo.svg">
```
[the remark this answers is not in the document]
Comment (Jokinen) [reply helsinki/login/template.ftl:64]: Hmn, my first thought was if I know how to do that, but I think I do.

I was told to "disregard" translations for now--but I think that you are still correct with this suggestion. I think that the expectations is that translations can be handled by adding message files in text which is not enough for the logo. I'll make this change.

File: sass/custom.scss (hunk 2 above)
Comment (Jokinen) [reply sass/custom.scss:123]: I'm thinking that the button component will be refactored along the lines of `TextInput` in which case its implementation details will likely change. But you are right that it pays to be explicit with these things. I can try and track down the cause in more depth to give some insight to the HDS team for when they take the button up for refactoring.

**Pull request merged**